canonical-kubernetes/steps/01_select-network/after-input

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -eux
+
+. "$CONJURE_UP_SPELLSDIR/sdk/common.sh"
+
+setKey "bundle-add" "$(scriptPath)/$NETWORKPLUGIN.yaml"
```

Comment:
So if we have a selection of items how do we know which fragment to remove? Setting the bundle-add/bundle-remove keys here would only work if a) there were only 2 options or b) we passed along the additional choices to the script so that we could match on what should not be included.

I think the scripts themselves may need to grow the ability to do the bundle modifications rather than conjure-up. @johnsca

Reply:
We only need to do any bundle modification if they choose the non-default option.  So, I was thinking something like this:

```bash
if [[ "$NETWORKPLUGIN" == "canal" ]]; then
  setKey "bundle-remove" "flannel.yaml"
  setKey "bundle-add" "canal.yaml"
fi
```

We should probably also assume paths relative to the step.